content/_data/changelogs/weekly.yml

```
@@ -15089,6 +15089,69 @@
   # pull: 6433 (PR title: Fix typo in dragdrop-debug.js)
   # pull: 6436 (PR title: Bump junit from 1.57 to 1.58)
 
+  - version: '2.343'
+    date: 2022-04-08
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 4579
+        issue: 50211
+        authors:
+          - stellargo
+          - basil
+        pr_title: "[JENKINS-50211] Reject connections from agents with unsupported\
+          \ Remoting versions"
+        message: |-
+          Reject connections from agents with unsupported Remoting versions.
+      - type: rfe
+        category: rfe
+        pull: 5459
+        issue: 40700
+        authors:
+          - Aki-7
+          - oleg-nenashev
+          - basil
+        pr_title: "[FIX JENKINS-40700] Display communication protocol in agent logs"
+        message: |-
+          Display communication protocol in agent logs
+      - type: major bug
+        category: regression
+        pull: 6447
+        issue: 67237
+        authors:
+          - MarkEWaite
+        pr_title: Revert "[JENKINS-67237] `BuildTrigger` waits until the dependency
+          graph has been updated (#6101)"
+        message: |-
+          [JENKINS-68189] Run downstream jobs (regression in 2.341).
+      - type: bug
+        category: bug
+        pull: 6435
+        issue: 68188
+        authors:
+          - benebsiny
+        pr_title: "[JENKINS-68188] Preview feature of <f:textarea> isn't disabled\
+          \ when readOnlyMode is set"
+        message: |-
+          Hide textarea preview when readOnlyMode is set
```

Needs a full stop at end of line
```suggestion
          Hide textarea preview when readOnlyMode is set.
```